users/vieting/jobs/fairseq_labels.py

```diff
@@ -0,0 +1,252 @@
+import soundfile
+import glob
+import os
+import random
+import numpy as np
+import shutil
+import subprocess
+
+from i6_core.lib import corpus
+
+from sisyphus import Job, Task, tk
+
+class FairseqFinetuneCorpusCreationJob(Job):
+    """
+    Creates a corpus directory for finetuning wav2vec with fairseq. This includes the following files:
+    - raw audios files
+    - transcriptions.txt
+    """
+    def __init__(self, corpus_path):
+        """
+        :param [tk.Path]|tk.Path corpus_path: list of paths or single path to corpus xml files.
+        """
+        if isinstance(corpus_path, tk.Path):
+            self.corpus_paths = [corpus_path]
+        else:
+            assert isinstance(corpus_path, list)
+            self.corpus_paths = corpus_path
```

Comment:
```suggestion
        if not isinstance(corpus_path, list):
            corpus_path = [corpus_path]
        self.corpus_paths = corpus_path
```
I guess it's debatable if we should rename `corpus_path` to `corpus_paths` from the beginning then.